content/docs/04.5-devx/07-palette-virtual-clusters/00-pause-restore-virtual-clusters.md

```diff
@@ -36,3 +36,10 @@ Invoke the pause and resume operations from the Palette Console.
 
 
 4. Click **Settings** and select the **Pause** option. To resume a paused virtual cluster, select the **Resume** option.
+
+# Validate the Pause or Resume Status of a Palette Virtual Cluster
```

Comment:
@resmaa this section is taking a lot of assumptions, such as assuming the user is already at the cluster page or in the proper context.  Let's avoid making assumptions. 
The validation section should start with a short intro sentence and then instruct the user on how to validate the state of the cluster.

Observe the following example.

 ## Validation

You can verify the state of a cluster by reviewing the cluster details. To review the state of a cluster and its details, do the following steps.

First, navigate to the "Mani Menu" and click on **Virtual Clusters** . Next, click on the specific cluster you want to check the status. This will take you to the cluster detail page. On this page, look for a section titled **Status**. The status section displays the current state of the cluster.

Reply:
sure, done